tests/ops/op_math/test_exp.py

```diff
@@ -399,6 +405,34 @@ def test_simplify(self):
         assert qml.equal(new_op.base, qml.PauliX(0))
         assert new_op.coeff == 0.2
 
+    def test_simplify_s_prod(self):
+        """Tests that when simplification of the base results in an SProd,
+        the scalar is included in the coeff rather than the base"""
+        base = qml.s_prod(2, qml.op_sum(qml.PauliX(0), qml.PauliX(0)))
+        op = Exp(base, 3)
+        new_op = op.simplify()
+
+        assert qml.equal(new_op.base, qml.PauliX(0))
+        assert new_op.coeff == 12
+
+    def test_copy(self):
```

Comment:
Also add an assert to make sure there are really two different objects: 
`assert copied_op is not op`